ene-jun-2021/Ernesto Burciaga Aguirre/Practica2_EBA.java

```diff
@@ -0,0 +1,36 @@
+package parcial_1;
+
+public class Practica2_EBA {
+
+public static void main(String[] args) {
+	
+	int [] num = new int[20];
+	
+	System.out.println("Arreglo sin ordenar");
+		for (int x=0;x<num.length;x++)
+			  num[x] = (int) (Math.random()*50)+1;
+		for (int i=0; i<num.length; ++i) 
+	        System.out.print("[" + num[i] + "]");
+		
+		for (int i = 1; i < num.length; i++) {
+			
+			int actual = num[i];
+	
+			int j = i-1;
+		for(; j >= 0; j--) {
```

Comment:
Cuida por acá el nivel de indentación, parece que este `for` está afuera del de la linea 15 :crying_cat_face: